### PR TITLE
Added KRW to Usd service

### PR DESCRIPTION
**Pull request by mrleewong**

### CODE REVIEW

File: AppConfig.js [hunk 1]
```diff
@@ -91,6 +91,11 @@ var config = {
         name: 'Yahoo',
```
Comment (shumkov) [inline AppConfig.js]: Don't we need `yahoofinance` anymore? Should we remove it?

Comment (mrleewong) [reply AppConfig.js]: Fixed

File: AppConfig.js [hunk 2]
```diff
@@ -87,10 +87,10 @@ var config = {
         urlBTC: 'https://cex.io/api/last_price/DASH/BTC',
         urlEur: 'https://cex.io/api/last_price/DASH/EUR'
       },
-      yahoofinance: {
-        name: 'Yahoo',
-        orgUrl: 'https://www.yahoo.com/',
-        url: 'https://query.yahooapis.com/v1/public/yql?q=select%20*%20from%20yahoo.finance.xchange%20where%20pair%20in%20(%22KRWUSD%22)&format=json&env=store://datatables.org/alltableswithkeys&callback='
+      currencyconverterapi: {
```
Comment (shumkov) [inline AppConfig.js]: Why don't you use camelCase? It's more readable.

Comment (mrleewong) [reply AppConfig.js]: fixed

**Comment by mrleewong** [comment]
@shumkov all good?

**Review by shumkov — APPROVED** [review]
utACK 👍

**Pull request merged**